packages/docs-ui/src/components/SideNavigation/SideNavigationSection.tsx

```diff
@@ -0,0 +1,50 @@
+import {
+  SideNavigationItem,
+  type SubNavigationItem,
+} from './SideNavigationItem';
+import { Box, HiddenVisually, Stack, Text } from 'braid-design-system';
+import * as styles from './SideNavigation.css';
+import React from 'react';
+
+interface SideNavigationSection {
+  title: string;
+  hideTitle?: boolean;
+  items: SubNavigationItem[];
+}
+
+export const SideNavigationSection = ({
+  title,
+  hideTitle,
+  items,
+}: SideNavigationSection) => (
+  <Box component="nav">
+    <Stack space="small">
+      <Box className={styles.uppercase}>
+        {hideTitle ? (
```

Comment:
Having the logic in here will still render the Box for a hidden title and subsequently the stack space as well. Might need to refactor this to only render the stack when the showing the title.